config.json

```diff
@@ -110,6 +111,15 @@
       "control-flow (case statements)",
       "control-flow (loops)"
     ]
+  }, {
+	"difficulty": 2,
```

Comment:
Whoever added the rna-transcription accidentally inserted a tab for that line (and I copied theirs). I'll fix them.